packages/compass-preferences-model/src/index.ts

```diff
@@ -50,6 +50,8 @@ export interface PreferencesAccess {
   createSandbox(): Promise<PreferencesAccess>;
 }
 export { setupPreferences };
-export const preferencesAccess: PreferencesAccess =
+export const defaultPreferencesInstance: PreferencesAccess =
```

Comment:
Super tiny nit, but I wonder if there is maybe a better name here, like `electronPreferencesInstance` or something, just to comminicate better the environment this is supposed to fully work in. I'm not best with naming so feel free to ignore completely 🙈

Reply:
Yeah, definitely up for changing the naming here if there’s something better, I’m not sure about `electronPreferencesInstance`, it’s not tied to Electron conceptually and is actually supposed to hide Electron-specific bits (such as the IPC variant) as an implementation detail?

This does work just fine in any Node.js environment, and could probably be easily adapted to work outside of even that?